js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties

```
@@ -3043,4 +3043,5 @@ enableClientSignatureRequiredModal=Enable client signature required
 selectBindType=Select bind type
 searchClientAuthorizationResource=Search resource
 searchClientAuthorizationPolicy=Search policy
-searchClientAuthorizationPermission=Search permission
\ No newline at end of file
+searchClientAuthorizationPermission=Search permission
+userNotSaved=The user hasn't been saved\: {{error}}
```

```suggestion
userNotSaved=The user has not been saved\: {{error}}
```

I don't know if this is a hard and fast rule, but we typically do not use contractions in an error message.   Contractions are considered less formal.

---

Thanks @ssilvert I can change this, but we also have two more messages using contractions, namely `updatedCredentialMoveError=User Credential configuration hasn't been saved` and `updatedUserProfileError=User Profile configuration hasn't been saved`

Would you like me to change them as well?